Workbooks/SapMonitor/SapHanaInfrastructure/SapHanaInfrastructure.workbook

```diff
@@ -1,17 +1,649 @@
 {
   "version": "Notebook/1.0",
   "items": [
+    {
+      "type": 11,
+      "content": {
+        "version": "LinkItem/1.0",
+        "style": "tabs",
+        "links": [
+          {
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Utilization",
+            "subTarget": "1",
+            "postText": "",
+            "style": "secondary"
+          },
+          {
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Anomalies",
+            "subTarget": "2",
+            "style": "secondary"
+          },
+          {
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Host Config",
+            "subTarget": "3",
+            "style": "secondary"
+          }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "testDataExist",
+        "comparison": "isNotEqualTo"
+      },
+      "name": "Tabs"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "query": "",
+        "crossComponentResources": [],
+        "parameters": [
+          {
+            "id": "8a72f4ec-ce64-49b7-88dc-d9b27de74f7a",
+            "version": "KqlParameterItem/1.0",
+            "name": "TimeRange",
+            "type": 4,
+            "value": {
+              "durationMs": 86400000
+            },
+            "typeSettings": {
+              "selectableValues": [
+                {
+                  "durationMs": 300000
+                },
+                {
+                  "durationMs": 900000
+                },
+                {
+                  "durationMs": 1800000
+                },
+                {
+                  "durationMs": 3600000
+                },
+                {
+                  "durationMs": 14400000
+                },
+                {
+                  "durationMs": 86400000
+                },
+                {
+                  "durationMs": 172800000
+                },
+                {
+                  "durationMs": 259200000
+                },
+                {
+                  "durationMs": 604800000
+                },
+                {
+                  "durationMs": 2592000000
+                },
+                {
+                  "durationMs": 5184000000
+                }
+              ]
+            },
+            "label": "Time range"
+          },
+          {
+            "id": "fae26122-3768-43dd-ac98-9b9768924799",
+            "version": "KqlParameterItem/1.0",
+            "name": "Granularity",
```

Comment:
if you don't want to force the user to do this to pick a grain, you can use `{TimeRange:grain}` in the query to get a "reasonable" default grain based on the selected time range

Reply:
Thanks, noted. I think for the time being we would expect users to specify the granularity (same behavior as SAP HANA Cockpit).